src/components/WeatherProperties/WeatherProperties.module.scss

```diff
@@ -1,5 +1,7 @@
 .weatherProperties {
   display: flex;
   flex-direction: column;
+  grid-area: properties;
```

Comment:
No GOD Noooooo

This is a universal component which has the only one goal is rendering properties from data which it receives from props, so it doesn't know anything about itself positioning

So you need to replace this positioning CSS to place which manage it, in our case it root Weazee component

Weazee component is a manager of positioning of its childs

Reply:
Why this code is a problem for this component? Because if you will use it in another place(for example on another page) you will have markup issues